install/0000_00_cluster-version-operator_03_deployment.yaml

```diff
@@ -36,6 +36,10 @@ spec:
         - "--serving-key-file=/etc/tls/serving-cert/tls.key"
         - "--v=2"
         - "--always-enable-capabilities=Ingress"
+        ports:
+        - name: http
```

Comment:
nit: can we call this `metrics` or something, so it's super-obvious what we're using it to serve?

Reply:
And whatever we call it, I think we can update [this Service reference][1] to use the new name, instead of the hardcoded port number?

[1]: https://github.com/openshift/cluster-version-operator/blob/d949cfe664794de94479f17627609bfeedf0ba53/install/0000_00_cluster-version-operator_04_service.yaml#L19

Reply:
- Removed `protocol: TCP` from the deployment as it is the default.
- Added `targetPort: metrics` to the service.